packages/api/resolvers/mutations/createProductBundleItem.js

```diff
@@ -1,16 +1,29 @@
 import { log } from 'meteor/unchained:core-logger';
-import { Products } from 'meteor/unchained:core-products';
-import { ProductNotFoundError, InvalidIdError } from '../../errors';
+import { Products, ProductTypes } from 'meteor/unchained:core-products';
+
+import {
+  ProductNotFoundError,
+  InvalidIdError,
+  ProductWrongTypeError,
+} from '../../errors';
 
 export default function createProductBundleItem(root, { productId, item }) {
   log(`mutation createProductBundleItem ${productId}`, { item });
+
   if (!productId) throw new InvalidIdError({ productId });
   if (!item.productId)
     throw new InvalidIdError({ bundleItemId: item.productId });
   const product = Products.findOne(productId);
-  if (!product) throw new ProductNotFoundError({ productId });
+
   const bundleProduct = Products.findOne(item.productId);
   if (!bundleProduct) throw new ProductNotFoundError({ productId });
+  if (!product) throw new ProductNotFoundError({ productId });
```

Comment:
The productId provided to the error here is not the item's productId. And also it looks like the bundleProduct is actually the item where as the product is the bundleProduct

Reply:
Maybe optimize the naming here along the way